### DISCUSSION TITLE
How to measure distance in meters between 2 points specified in longitude, latitude (WGS 84)?

### DISCUSSION THREAD

**Discussion by woutware** [DotSpatial/DotSpatial#1554] [Q&A]
Hi,

How do I measure distance in meters between 2 points specified in longitude, latitude (WGS 84)? The 2 points are relatively close together, within 10 km generally.

Also somewhat related, how do I get a projection to local cartesian that's centered around a point on earth (longitude, latitude)?

Thank you!

Wout

**Comment by sindizzy** [comment]
What I typically do is reproject to Equidistant Azimuthal centered in your area of interest. Then it's a simple length of the segment as the unit is meters.

Alternatively you could use a library designed for such things like GeoLib or Gavaghan.
